.gitignore

```diff
@@ -6,3 +6,4 @@ npm-debug.log
 _book
 lib
 _site
+.idea
```

Comment:
If you don't mind I'd skip this change, as it's not related to the source code :)